cmd/tarocli/proofs.go

```diff
@@ -28,6 +29,9 @@ var proofCommands = []cli.Command{
 
 const (
 	proofPathName = "proof_file"
+
+	proofIndexName = "proof_index"
+	allProofsName  = "all_proofs"
```

Comment:
This should probably match the RPC's name (`latest_proof`) since only one proof is validated.